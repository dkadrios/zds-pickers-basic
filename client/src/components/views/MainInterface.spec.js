import WrappedMainInterface from './MainInterface'

const MainInterface = WrappedMainInterface.DecoratedComponent

describe('MainInterface tests', () => {
  beforeEach(jest.clearAllMocks)

  it('renders correctly', () => {
    shallowExpect(MainInterface)({}).toMatchSnapshot()
  })
})
