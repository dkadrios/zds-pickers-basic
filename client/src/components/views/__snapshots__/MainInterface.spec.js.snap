// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainInterface tests renders correctly 1`] = `
<div>
  <Component
    backend={[Function]}
  >
    <PrimaryNav />
  </Component>
  <Connect(HardwareTest) />
  <Connect(MidiSecurity) />
  <Connect(NotResponding) />
  <Connect(VersionNotSupported) />
  <Connect(NoShifterFound) />
  <Connect(InfoPanel) />
</div>
`;
